pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "teslajsonpy"
-version = "0.21.0"
+version = "0.21.2"
```

Comment:
Revert the version change, semantic release will handle.

Reply:
done